src/mono/sample/wasm/browser2/Program.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Sample
+{
+    public class Test
+    {
+        public static void Main(string[] args)
```

Comment:
How does this relate to `App.init()` why it's not the same ?

Reply:
App.init is entirely JS whereas this Main method is the entry point into C#.

However perhaps this brings up a good point. Maybe `App.init` should be renamed to something else. I just left it to make it easier to see the differences between this new sample and the old one.

Reply:
I renamed it to SampleApp per your recommendation.